docs/advanced/Testing.md

```diff
@@ -1,9 +1,35 @@
 # Examples of Testing Sagas
 
-**WIP**
+There is ongoing discussion in [this issue](https://github.com/redux-saga/redux-saga/issues/518) about the best way to test Sagas.
 
-See Repository Examples:
+
+In the repo we also have two examples showing how Sagas can be tested:
 
 https://github.com/redux-saga/redux-saga/blob/master/examples/counter/test/sagas.js
 
 https://github.com/redux-saga/redux-saga/blob/master/examples/shopping-cart/test/sagas.js
+
+
+
+***External Resources***
+
+Redux Saga Test Plan - https://github.com/jfairbank/redux-saga-test-plan
+
+From the README:
```

Comment:
For sure we would like to have some official stand for testing library and possibly include it in the organization, however its not yet time for this.

The primary goal for now is to reach API stability (mostly done), fix some bugs (minor ones), add some missing features (ideas are welcome!) and ship v1 with all mentioned things. 

Then I think the testing story will get one of the highest priority. I wouldnt like yet to favor one of the approaches over another (which taking over the repo kinda is), as I didnt rly have time to gather my own thoughts about testing properly and also didnt evaluate thoroughly available helper libraries (such as yours) to see what approach did they chose and how they differ.